Libraries/Template/Components/Header.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { View, Text, StyleSheet, ImageBackground } from 'react-native'
+
+const Header = () => (
+  <View style={styles.container}>
+    <ImageBackground
+      source={require('../logo.png')}
+      style={styles.backgroundLogo}
+    />
+
+    <Text style={styles.text}>Welcome to React Native</Text>
+  </View>
+)
+
+const styles = StyleSheet.create({
+  container: {
+    height: 270,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: '#F3F3F3',
```

Comment:
What do you think of extracting all those colors and use it across the template files?

```
const Colors =  {
  black: "#000",
  ....
}

const styles = StyleSheet.create({
  container: {
    color: Colors.black,
    ...
  }
}
```
It's just a suggestion to demonstrate style reusability for newcomers